docs/source/scenarios/vm_scheduling.rst

```diff
@@ -67,15 +67,28 @@ There are two types of meaningful actions:
 
 See the detailed attributes of `Action <#id1>`_.
 
+
+Oversubscription
+~~~~~~~~~~~~~~~~~~~~
+To maximize each PM's utilization, cloud providers will oversubscribe the physical resource.
+The oversubscription rate can be set in the config.yml. In our scenario, there are two resources
+could be oversubscribed, CPU and memory.
+
+* ``MAX_CPU_OVERSUBSCRIPTION_RATE``: The oversubscription rate of CPU.
+* ``MAX_MEM_OVERSUBSCRIPTION_RATE``: The oversubscription rate of memory.
+
+For example, the default setting of ``MAX_CPU_OVERSUBSCRIPTION_RATE`` is 1.15, that means each
+PM can be allocated at most 1.15 times of its resource capacity.
+
```

Comment:
Considering the various service level, the physical machines are then divided into the ones over-subscribable and ...

Reply:
Shall we also introduce about the VM category (PM will be non-oversubscribable if one of its vm is interactive)?

Reply:
Good point, introduce it in `VM Request` part?